IdentifiabilityODE.mpl

```diff
@@ -351,12 +351,21 @@ IdentifiabilityODE := proc(system_ODEs, params_to_assess, {p := 0.99, infolevel
     printf("%s %a\n", `Not identifiable parameters:                      `, map(x -> ParamToOuter(x, all_vars), select(p -> not p in theta_l, theta))):
     printf("===============\n\n"):
   end if:
-
-  table([
+  output := table([
     globally = {op(map(x -> ParamToOuter(x, all_vars), theta_g))},
     locally_not_globally = {op(map(x -> ParamToOuter(x, all_vars), select(p -> not p in theta_g, theta_l)))},
     non_identifiable = {op(map(x -> ParamToOuter(x, all_vars), select(p -> not p in theta_l, theta)))}
   ]):
+  for var in map(ParamToInner, output[globally]) do
+ 	  printf("%s %a %s %a\n",`The number of solutions for`, var, `is`, 1):
+  end do:
+	
+  for var in map(ParamToInner, output[locally_not_globally]) do
+	  G := Groebner[Walk](gb, tdeg(op(vars)), lexdeg([op({op(vars)} minus {var})], [var])):

```

Comment:
This `gb` will depend on the methods used (1 or 2). In particular, this code is for method 2. So I think it should be together  with method 2, in lines 337-343.

Reply:
moved to the section right before method 3